cli/src/main/java/org/openjdk/skara/cli/GitJCheck.java

```diff
@@ -349,6 +349,9 @@ else if (confFile || confWorkingTree || workingTree) {
                 }
             } catch (Exception e) {
                 System.err.println(String.format("error: exception thrown during jcheck: %s", e.getMessage()));
+                if (e.getMessage().equals("java.net.ConnectException")) {
```

Comment:
I forgot to look at the rest of the code. I think this should be an `instanceof` check instead of comparing strings.

Reply:
Sure